tools/wsdl2aws-generator-stub.adb

```diff
@@ -488,6 +496,14 @@ package body Stub is
          Text_IO.New_Line (Stub_Adb);
       end if;
 
+      if O.Gen_CB then
```

Comment:
I'm not sure we want to have this feature tied to the -cb option. We can probably want to be able to get traces without having generated the the SOAP callbacks. The traces are in the client part of the code. Maybe a new option -traces would be better. Do you agree?

Reply:
I agree, but was too lazy to figure out how to add a new option.  I'll take a look at it.